api-schema/src/main/java/org/stellar/anchor/api/sep/sep6/RefundPayment.java

```diff
@@ -0,0 +1,17 @@
+package org.stellar.anchor.api.sep.sep6;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+@Data
+public class RefundPayment {
```

Comment:
We have the class: `org.stellar.anchor.api.shared.RefundPayment`. Can we re-use?

Reply:
Created ANCHOR-392 to use the shared type in SEP-24 and SEP-31 implementations.